date-time/src/commonMain/kotlin/com/splendo/kaluga/datetime/timer/Timer.kt

```diff
@@ -28,7 +27,9 @@ interface Timer {
     /** Timer duration. */
     val duration: Duration
     /** Current state of the timer. */
-    val state: StateFlow<State>
+    val state: Flow<State>
+    /** The Default [State] of the timer. */
+    val defaultState: State
```

Comment:
Timer states are running/notrunning, for me default state is always NotRunning as we can't create already running timer with the given offset (for examples running timer from 00:10 and on)